packages/appsync-modelgen-plugin/src/plugin.ts

```diff
@@ -59,3 +64,11 @@ export const plugin: PluginFunction<RawAppSyncModelConfig> = (
   }
   return '';
 };
+
+export const plugin: PluginFunction<RawAppSyncModelConfig> = (
```

Comment:
Add docs to make clear how its okay that async is using sync. Make clear that sync version is called from async signature.

Reply:
Added